### PR TITLE
clean up Sync Data button

### PR DESCRIPTION
**Pull request by benrudolph**
@wpride this cleans up the Sync Data button. was playing around with it a bit. it now disables after click and shows loading.
screen shot 2016-01-06 at 10 05 47 pm
screen shot 2016-01-06 at 10 04 41 pm
buddy: @TylerSheffels

### CODE REVIEW

File: corehq/apps/style/templates/style/bootstrap3/two_column.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {% block page_breadcrumbs %}{% endblock %}
     <div class="container-fluid">
         <div class="row">
-            <div id="hq-sidebar" class="col-xs-4 col-sm-3 col-lg-2 col-hq-sidebar">
+            <div id="hq-sidebar" class="col-xs-4 col-sm-3 col-lg-2 col-hq-sidebar nav-hq-sidebar">
```
Comment (benrudolph) [inline corehq/apps/style/templates/style/bootstrap3/two_column.html]: @biyeun any reason not to add this class? it allowed me to use `nav-input`

**Comment by wpride** [comment]
this is tight! :smoking: :fire: :tight: :moneybag:

**Comment by TylerSheffels** [comment]
Looks good to me

**Comment by benrudolph** [comment]
:white_check_mark:

**Pull request merged**